### ISSUE TITLE
autopilot.js: BN 10 - enhancements to help with purchasing additional sleeves

### ISSUE THREAD

**Issue by alainbryden** [alainbryden/bitburner-scripts#373]
BN10 will not complete if the user does not have the maximum number of sleeves.

When this code was written, there was no API for purchasing or upgrading sleeves. We should review whether this is still the case.

Regardless, we should see if there's anything we can do in this BN to focus on earning more money over any other concerns, since we need an incredible amount (100q?) to purchase them all and upgrade their memory. IIRC When I played this, I relied on corps to gain this much money, not sure if it's possible to automate getting this much money without adding corps scripts.

**Comment by mikeymikey64** [comment]
perhaps it can be made so that it tells the player to leave early? since the amount of money needed is so high, I left manually before having all sleeves and plan to revisit it after other money-making mechanics open up.

**Comment by alainbryden** [comment]
@mikeymikey64 I tried a fresh run on a new game save.

When I finally reached 10.1, it took 31 hours to "beat" the bitnode, and only 18 additional hours (49 hours total) in BN10.1 to reach > 100q (money required to purchase the 6th sleeve).

Daemon.js' normal batcher script is sufficient to reach this amount of money quickly once all augmentations are installed and NF reaches ~LVL 120. So I think it's sensible to recommend the player stick around in the BN the first time around until they can max their sleeves.
